src/mcts/search.cc

```diff
@@ -1216,11 +1216,41 @@ void SearchWorker::GatherMinibatch() {
     if (search_->stop_.load(std::memory_order_acquire)) return;
   }
 }
+
+namespace {
+int Mix(int high, int low, float ratio) {
+  return static_cast<int>(std::round(high * ratio + low * (1.0f - ratio)));
+}
+int CalculateCollisionsLeft(int64_t nodes, const SearchParams& params) {
+  // End checked first
+  if (nodes >= params.GetMaxCollisionVisitsScalingEnd()) {
+    return params.GetMaxCollisionVisitsId();
```

Comment:
I think `GetMaxCollisionVisitsId()` should not have `Id` in its name. :)

Reply:
Yeah - pre-existing condition though :P

Reply:
Fixed.